test/k8sT/manifests/test-verifier.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-verifier
+spec:
+  containers:
+  - name: cilium-builder
+    image: quay.io/cilium/cilium-builder:2020-10-20
```

Comment:
Maybe we should switch this to use `quay.io/cilium/cilium-builder:2020-10-15-v1.8@sha256:5f2e2e4cb4b86525f23cc183ffabb0297060b269f830377a548ea60c058951f9`, i.e., the tag we use in v1.8?

Reply:
Good point, done (and updated to point to the newly built `quay.io/cilium/cilium-builder:2020-11-09-v1.8@sha256:660cb6ec90c0e662b710a86ab876ecf1a1d0a503269c6751ef3d2c4afc18666e` tag)